docker/pravega/entrypoint.sh

```diff
@@ -112,6 +112,12 @@ configure_standalone() {
     add_system_property "singlenode.zkPort" "2181"
     add_system_property "singlenode.controllerPort" "9090"
     add_system_property "singlenode.segmentstorePort" "12345"
+    add_system_property "singlenode.certFile" "./config/cert.pem"
+    add_system_property "singlenode.keyFile" "./config/key.pem"
+    add_system_property "singlenode.passwdFile" "./config/passwd"
```

Comment:
Could these config paths be made more reliable by basing them on the script path?  e.g. ([ref](https://stackoverflow.com/a/246128/3026310))
```
BINDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
add_system_property "singlenode.certFile" "${BINDIR}/../config/cert.pem"
```

Otherwise we're assuming that the current directory is the installation root.

Reply:
Maybe the default should be an insecure cluster, and avoid baking in these secrets.